src/patternfly/components/Label/label.scss

```diff
@@ -462,6 +454,22 @@
   color: var(--pf-c-label__icon--Color);
 }
 
+.pf-c-label__actions {
+  margin-right: var(--pf-c-label-actions--MarginRight);
+  font-size: var(--pf-c-label__actions--FontSize);
+
+  & .pf-c-button {
+    --pf-c-button--FontSize: var(--pf-c-label-actions--c-button--FontSize);
+    --pf-c-button--PaddingTop: var(--pf-c-label-actions--c-button--PaddingTop);
+    --pf-c-button--PaddingRight: var(--pf-c-label-actions--c-button--PaddingRight);
+    --pf-c-button--PaddingBottom: var(--pf-c-label-actions--c-button--PaddingBottom);
+    --pf-c-button--PaddingLeft: var(--pf-c-label-actions--c-button--PaddingLeft);
+
+    margin-top: var(--pf-c-label-actions--c-button--MarginTop);
+    margin-bottom: var(--pf-c-label-actions--c-button--MarginBottom);
```

Comment:
do you think it's beneficial to keep these on the button? I'm thinking whatever actions we have in here will all have the same padding, so it might be easier just to put it on the actions container?

Reply:
Since the padding is on the button (and it needs to be for the clickable area) I think it's best to keep the negative margin on the same element.